packages/jitm/src/JITM.php

```diff
@@ -80,6 +81,9 @@ function prepare_jitms( $screen ) {
 			add_action( 'admin_enqueue_scripts', array( $this, 'jitm_enqueue_files' ) );
 			add_action( 'admin_notices', array( $this, 'ajax_message' ) );
 			add_action( 'edit_form_top', array( $this, 'ajax_message' ) );
+
+			// Not really a JITM. Don't know where else to put this :)
```

Comment:
help :)

Reply:
As a dev, I'd probably search for something like this from `packages/connection` but not sure how this plays with that package needing to be more generic and not Jetpack-specific?

Reply:
We decided at one point to let UI decisions be made by the consumer, and not include UI into the connection package. We can decide the other way later, but it's not important enough to block this PR over IMO.

Reply:
Makes total sense! 👍